src/Compilers/CSharp/Portable/Symbols/ParameterSymbol.cs

```diff
@@ -232,6 +232,14 @@ public bool HasExplicitDefaultValue
             }
         }
 
+        /// <summary>
+        /// Returns the bound default value syntax from a parameter, if it exists.
+        /// Note that this field will only be non-null if a default parameter value was 
+        /// supplied in syntax and not in metadata. Thus, there are cases
+        /// where ExplicitDefaultValue will be non-null, but this field will be.
+        /// </summary>
+        public virtual BoundParameterEqualsValue? BoundEqualsValue { get { return null; } }
```

Comment:
Thinking more about this, I'm concerned about a symbol holding onto a BoundNode of any kind. It feels likely that this is going to cause some issues with various rewriting passes in some manner. @cston, what do you think?

Reply:
Holding onto BoundNodes might affect rewriting. Also, symbols (outside of method bodies) should not hold onto SyntaxNodes (which BoundNodes do) because that will prevent SyntaxTrees from being garbage collected.